src/NzbDrone.Core/MediaFiles/FileExtensions.cs

```diff
@@ -3,10 +3,11 @@
 
 namespace NzbDrone.Core.MediaFiles
 {
-    internal static class FileExtensions
+    public static class FileExtensions
     {
         private static List<string> _archiveExtensions = new List<string>
         {
+            ".001",
```

Comment:
I think for the time being we shouldn't include it as an archive file (probably it's being used for junk/malicious files) based on some reports, but I wouldn't use it in the example.

Reply:
removed this commit. makes sense.